go.mod

```diff
@@ -25,3 +22,5 @@ require (
 	sigs.k8s.io/controller-tools v0.2.9-0.20200331153640-3c5446d407dd
 	sigs.k8s.io/yaml v1.2.0
 )
+
+replace github.com/openshift/machine-api-operator => github.com/joelspeed/machine-api-operator v0.2.1-0.20200417102748-367ae647375f
```

Comment:
Is this intentional that it's pointing at a repository in the `joelspeed` namespace rather than in the `openshift` namespace?

Reply:
I meant to change it back before this got merged. I'll raise a PR to put it back.